src/app.js

```diff
@@ -0,0 +1,77 @@
+const renderApp = () => {
+  const calc = document.querySelector('.calc');
+
+  const createComponent = (
+    {
+      tagName,
+      className,
+      value,
+      onClick,
+      parentNode,
+    },
+  ) => {
+    if (!tagName) { throw new Error('Component should have tagName'); }
+    const node = document.createElement(tagName);
+    if (className) { node.className = className; }
+    if (value) { node.textContent = value; }
+    if (parentNode) { parentNode.append(node); }
+    if (onClick) { node.onclick = onClick; }
+    return node;
+  };
+
+  const calcScreen = createComponent({
+    tagName: 'div',
+    className: 'calc-screen',
+    parentNode: calc,
+  });
+
+  // eslint-disable-next-line no-unused-vars
+  const output = createComponent({
```

Comment:
```suggestion
  createComponent({
```

you should use `eslint-disable-next-line no-unused-vars`